A:
  ThetaX: -0.1870432049036026
  ThetaY: 0.7299202084541321
  ThetaZ: 2.315800905227661
  X: 0.5330418348312378
  Y: -0.16477394104003906
  Z: 0.6382796764373779
B:
  ThetaX: -0.39665132761001587
  ThetaY: 0.6543828248977661
  ThetaZ: 2.6411798000335693
  X: 0.37658143043518066
  Y: -0.01368001103401184
  Z: 0.8633081316947937
test:
  ThetaX: -0.19061850011348724
  ThetaY: 0.7474631071090698
  ThetaZ: 2.301758050918579
  X: 0.3918577790260315
  Y: -0.12015067785978317
  Z: 0.639392614364624
